ext/QuantumSavoryMakie/QuantumSavoryMakie.jl

```diff
@@ -280,27 +280,44 @@ end
 
 ##
 
-"""Draw the given registers on a given Makie axis.
```

Comment:
Does this not support subfigures anymore? That would be a breaking change. 

But reading through the rest of the code, it seems subfigures are still supported, so I guess it is just incorrect edit to the docstring.

Reply:
nope, forget this comment, I just got confused